lib/chef/resource/aws_rds_instance.rb

```diff
@@ -0,0 +1,33 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+
+class Chef::Resource::AwsRdsInstance < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type AWS::RDS::DBInstance
+
+  attribute :db_instance_identifier, kind_of: String, name_attribute: true
```

Comment:
Awesome - this is unique across the account and region.  That means we don't have to create this class as a `Chef::Provisioning::AWSDriver::AWSResourceWithEntry`